### PR TITLE
Case-insensitive checking string in string list

### PR DESCRIPTION
**Pull request by fivard**

### CODE REVIEW

File: json_logic/__init__.py [hunk 1]
```diff
@@ -100,6 +100,9 @@ def merge(*args):
 def in_operator(a, b):
     """Implements the 'in' operator for work with lists."""
     if "__contains__" in dir(b):
+        if isinstance(a, str):
+            # case-insensitive checking string in string list
+            return a.casefold() in [string.casefold() for string in b if isinstance(string, str)]
```
Comment (hoherchak) [inline json_logic/__init__.py:105]: What if the list has non-strings? Please cover with tests properly considering various cases.

**Review by hoherchak — COMMENTED** [review]
Please add same for `==` operator as we touch this anyway.

File: json_logic/__init__.py (hunk 1 above)
Comment (fivard) [reply json_logic/__init__.py:105]: It is checked in `if` condition in the end of lambda expression

Comment (fivard) [reply json_logic/__init__.py:105]: Will add to `==`

**Review by hoherchak — APPROVED** [review]
Sounds good

**Pull request merged**